index.css

```
@@ -244,6 +252,272 @@
+    justify-content: center;
+    align-items: center;
+    margin-top: 90px;
+  }
+
+  .worksHeadInfo {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: flex-start;
+  }
+
+  .worksHeadInfo h3 {
+    font-family: 'Crete Round', serif;
+    font-weight: 400;
+    font-size: 32px;
+    color: #091e42;
+    line-height: 44px;
+    margin: 24px 0 12px 0;
+  }
+
+  .worksHeadInfo p {
+    font-family: 'Inter', sans-serif;
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 24px;
+    color: #091e42;
+  }
+
+  .tagGroup {
+    margin: 12px 0;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    font-family: 'Inter', sans-serif;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #091e42;
+  }
+
+  .tagGroup div {
+    padding: 10px 12px;
+    border: 1px solid #8993a4;
+  }
+
+  .projectBtn {
+    margin-bottom: 24px;
+  }
+
+  .projectBtn button {
+    color: white;
+    font-family: 'Inter', sans-serif;
+    font-weight: 700;
+    font-size: 17px;
+    line-height: 24px;
+    background-color: #ff6b00;
+    padding: 12px;
+    border: none;
+  }
+
+  .container {
+    display: grid;
+    grid-template: repeat(6, 1fr) / 1fr;
+    row-gap: 24px;
+    margin-bottom: 120px;
+  }
+
+  .container .card {
+    height: 386px;
+    background-image: url(assets/img/cardBack.png);
+    background-repeat: no-repeat;
+    background-position: center;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+    align-items: flex-start;
+  }
+
+  .card .info {
+    padding: 0 24px;
+  }
+
+  .info h3 {
+    color: white;
+    font-family: 'Crete Round', serif;
+    font-weight: 400;
+    font-size: 32px;
+    line-height: 44px;
+  }
+
+  .info p {
+    color: white;
+    font-family: 'Inter', sans-serif;
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 24px;
+  }
+
+  .card .cardTagGroup {
+    margin: 12px 24px 24px 24px;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: flex-start;
+    font-family: 'Inter', sans-serif;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: white;
+  }
+
+  .cardTagGroup div {
+    margin-right: 12px;
+    padding: 10px 12px;
+    background: rgba(255, 255, 255, 0.24);
+  }
+
+  .card .bottomBtn {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #ff6b00;
+    height: 48px;
+    width: 100%;
+  }
+
+  .bottomBtn button {
+    color: white;
+    font-family: 'Inter', sans-serif;
+    font-weight: 700;
+    font-size: 17px;
+    line-height: 24px;
+    background-color: #ff6b00;
+    padding: 12px;
+    border: none;
+  }
+
+  .about {
+    padding: 0 24px;
+    display: flex;
+    flex-direction: column;
+    background-color: #1c1a19;
+    background-image: url(assets/img/Frame.png);
+    background-repeat: no-repeat;
+    background-position-x: 215px;
```

[OPTIONAL] 

- [ ] You can set this value to `right`, and the background will stick to the right on bigger mobile resolutions also. 

```suggestion
    background-position-x: right;
```

---

This issue is successfully addressed